app/views/stafftools/users/_user.html.erb

```diff
@@ -5,5 +5,8 @@
     <strong class="search-result-title"><%= user.login %></strong>
   <% end %>
 
-  <span class="text-muted css-truncate css-truncate-target">Joined <%= local_time_ago(user.created_at) %></span><br>
+  <span class="text-muted css-truncate css-truncate-target">
+    Joined <%= local_time_ago(user.created_at) %> /
+    Last active: <%= user.last_active_at.present? ? local_time_ago(user.last_active_at) : 'Unknown' %>
```

Comment:
Yes, this should not be null in our main Postgres database.

I was wondering if there is a case that the indexes on Elasticsearch is corrupted/outdated, which might cause this value to be null.